src/modules/chat_highlight_blacklist_keywords/index.js

```diff
@@ -198,6 +202,13 @@ class ChatHighlightBlacklistKeywordsModule {
             defaultValue: false
         });
 
+        settings.add({
```

Comment:
this is not something we should make a separate setting. instead, it should be made possible to add a match for this in the existing highlights system